packages/puppeteer-core/src/cdp/Accessibility.ts

```diff
@@ -129,8 +129,9 @@ export interface SnapshotOptions {
  * the "interesting" nodes of the tree.
  *
  * @privateRemarks BiDi
- * Not supported when using protocol `webDriverBiDi`
- * Support for BiDi+
+ * {@link PROTOCOL_GET_STARTED_LINK_TEMPLATE | CDP}: Supported
```

Comment:
why is this one without a line break in-between and with BiDi+?

Reply:
Just missed it.